repytah/search.py

```diff
@@ -180,8 +180,8 @@ def find_complete_list(pair_list, song_length):
 def __find_add_rows(lst_no_anno, check_inds, k): 
     """
     Finds pairs of repeated structures, represented as diagonals of a certain
-    length, k, that neither start nor end at the same time steps as previously
-    found pairs of repeated structures of the same length.
+    length, k, that start at, end at, or in thr middle of the same time steps
+    as previously found pairs of repeated structures of the same length.
```

Comment:
Probably a longer but (hopefully) clearer version might help: "... that start at the same time steps, or end at the same time steps, or neither start nor end at the same time step..." (basically the previous description of the 3 functions combined). It feels clearer that way but maybe it's just me.

Reply:
Also "the" is mispronounced